xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.h

```diff
@@ -59,6 +61,7 @@ class CActiveAESound : public IAESound
   static int64_t Seek(void *h, int64_t pos, int whence);
 
 protected:
+  CActiveAE *m_activeAE;
```

Comment:
I did not speak of AE as a whole, I commented on this specific implementation and as I mentioned before, a circular dependency is basically never a valid design, this does not in any way depend on how good the overall code is. 

I also mentioned that I consider it acceptable if it's part of a redesign where the end goal gets rid of it as that can be necessary. 

I don't see why it matters who wrote this and the surrounding code. Are you trying to say that only the person who wrote the code is allowed to comment? That to me does not seem like a good way to work as a team and I'd rather avoid it.